src/features/licenses/components/LicenseForm/__test__/index.test.jsx

```diff
@@ -140,7 +145,33 @@ describe('LicenseForm component', () => {
       <LicenseForm
         created={false}
         errors={{}}
-        setFields={() => {}}
+        setFields={() => { }}
+        fields={formValues}
+      />,
+      { preloadedState: mockStore },
+    );
+
+    // Ensure that the selected catalog is displayed
+    expect(getByText('Catalogs')).toBeInTheDocument();
+    expect(getByText('full catalog')).toBeInTheDocument();
+  });
+
+  test('Edit mode with multipe catalogs', () => {
```

Comment:
How suppose this test the different scenarios?
1. Using the catalog as license name
2. Using a custom license name
3. edit the custom catalog without change the license name
4. edit the custom catalog including the license name

Reply:
I updated tests, the edit mode is not modified in this scope, so I don't added them